statistics/table.go

```diff
@@ -688,9 +690,17 @@ func (coll *HistColl) GetAvgRowSize(cols []*expression.Column, isEncodedKey bool
 			}
 			// We differentiate if the column is encoded as key or value, because the resulted size
 			// is different.
-			size += colHist.AvgColSize(coll.Count, isEncodedKey)
+			if sessionVars.EnableChunkRPC {
+				size += colHist.AvgColSizeChunkFormat(coll.Count)
```

Comment:
If a cost is describing scan factor in TiKV, should we always use `colHist.AvgColSize` but not `colHist.AvgColSizeChunkFormat`?
Like https://github.com/pingcap/tidb/pull/13461/files#diff-152a0fbc96916798e8bccdb90988517fR689

Reply:
Addressed. PTAL again. @SunRunAway